nuxt/index.js

```diff
@@ -0,0 +1,26 @@
+export default function vueSvgInlineLoaderModule(moduleOptions) {
+	const options = Object.assign(
+		{},
+		this.options.vueSvgInlineLoader,
+		moduleOptions
+	);
+
+	this.extendBuild((config) => {
+		const vueRule = config.module.rules.find(
+			({ test }) => test.toString() === /\.vue$/i.toString()
+		);
+		vueRule.use = [
+			{
+				loader: vueRule.loader,
+				options: vueRule.options,
+			},
+			{
+				loader: join(__dirname, "../index.js"),
+				loader: "vue-svg-inline-loader",
```

Comment:
Duplicate `loader` key - this is definitely not correct. First definition should be correct.

Reply:
My bad, one was commented like in the old version. I'll fix that